src/js/Features.js

```diff
@@ -37,7 +38,19 @@ const Features = function (config) {
         );
     }
 
-    self._features = features.sort((a, b) => a.name.localeCompare(b.name, window.navigator.language, { ignorePunctuation: true }));
+    self._features = features;
+
+    if (semver.gte(config.apiVersion, API_VERSION_1_45) && FC.CONFIG.buildKey) {
+        const filter = ["GPS", "DASHBOARD", "LED_STRIP", "OSD", "RANGEFINDER", "SERIAL_RX", "SERVOS", "SPI_RX", "TELEMETRY", "TRANSPONDER"];
```

Comment:
I think we are having too much hardcoded elements. This will require to maintain in coherence firmware/configurator. I understand to do this for features, that is something that we are trying to remove so it will not grow, but I don't know if we are planning to do the same for other UI elements.

Reply:
Yes this is hack, becomes the 'other' group contains also non-features. While features are very stable over time this PR is just enabling support for cloud parameters AND applying this to features as a demonstration.

Reply:
Have removed the hack and added a property to the feature instead.